utils/scripts/createDB.js

```diff
@@ -1,3 +1,2 @@
 const db = require('../database');
-require('dotenv').config();
-console.log("Created database with errors: \n" + db.createSchema());
\ No newline at end of file
+require('dotenv').config();
```

Comment:
you removed the db.createSchema() part from this script.. add it back in.

"console.log("Created database with errors: \n" + db.createSchema());"